src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HandleWrapper.cpp

```diff
@@ -4,5 +4,6 @@
 #include "HandleWrapper.h"
 
 // Workaround for VC++ bug https://developercommunity.visualstudio.com/content/problem/33928/constexpr-failing-on-nullptr-v141-compiler-regress.html
+// REVIEW: This is fixed maybe? We could try removing it now that we're on v142. Is there test coverage?
```

Comment:
If this is fixed yeah. I think it will fail to build if this isn't fixed, so we have "test coverage"

Reply:
Yep, I realized it was literally a compiler bug, so it felt safe to try :)